actors/user/src/main/java/org/sunbird/user/actors/UserManagementActor.java

```diff
@@ -60,21 +64,20 @@
  * @author Amit Kumar
  */
 @ActorConfig(
-  tasks = {
-    "createUser",
-    "updateUser",
-    "getUserProfile",
-    "getRoles",
-    "getUserDetailsByLoginId",
-    "profileVisibility",
-    "unblockUser",
-    "blockUser",
-    "assignRoles",
-    "userCurrentLogin",
-    "getMediaTypes"
-  },
-  asyncTasks = {}
-)
+    tasks = {
+      "createUser",
+      "updateUser",
+      "getUserProfile",
+      "getRoles",
+      "getUserDetailsByLoginId",
+      "profileVisibility",
+      "unblockUser",
+      "blockUser",
+      "assignRoles",
+      "userCurrentLogin",
+      "getMediaTypes"
+    },
+    asyncTasks = {})
 public class UserManagementActor extends BaseActor {
```

Comment:
Another PR is required which splits UserManagementActor into UserActor, UserStatusActor, UserLoginActor, and UserProfileActor.